WordPress-Extra/ruleset.xml

```diff
@@ -168,6 +168,11 @@
 	</rule>
 
 
+	<!-- Prevent some typical mistakes people make accidentally.
+	     .. -->
```

Comment:
This bit seems incomplete?

Reply:
Oh darn! Yes, I needed the PR number for that, but obviously didn't have it when I opened the PR.
And then I promptly forgot about it :joy: